src/config.rs

```diff
@@ -36,17 +36,25 @@ impl Default for Config {
 
 impl Config {
     /// Load the configuration file from either `path` or the default config dir
-    /// `${XDG_CONFIG_HOME}`.
+    /// `${XDG_CONFIG_HOME}` or `/etc/xdg`.
     ///
     /// Error: File does not exist or can't be opened, syntax errors and other parsing errors.
     pub fn load(path: Option<PathBuf>) -> Result<Self, Error> {
         let cfg_path: PathBuf = if let Some(p) = path {
             p
         } else {
-            dirs::config_dir()
+            let user_path = dirs::config_dir()
                 .ok_or_else(|| format_err!("Can't access default config dir."))?
```

Comment:
So this will fail if there is no `config_dir` but the sys dir exists right? How about using the `sys_path` in the `ok_or_else`?

E.g., something like this:

```rust
            let sys_path = PathBuf::from("/etc/xdg/");
            dirs::config_dir()
                .unwrap_or(sys_path)
                .join(env!("CARGO_PKG_NAME"))
                .join("config.toml")
```